package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bat-client",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
should this be `3.0.0` since it is a breaking api change?

Reply:
good catch, changed